addons/web/static/src/views/kanban/kanban_renderer.js

```diff
@@ -541,6 +542,7 @@ export class KanbanRenderer extends Component {
      */
     sortStart({ element }) {
         element.classList.add("o_dragged", "shadow");
+        this.state.isResequenceInprogress = true;
```

Comment:
this triggers a re-rendering of the kanban_renderer each time the user starts dragging a record or a column, I'm not sure if this is good idea/if this could lead to issues. Also, I think this task a very low added value. I wouldn't do it personally...

Reply:
I added some extra condition to avoid re-rendering when it is not needed to hide no content helper :slightly_smiling_face:

Reply:
So it's more complexity bug still little low value... @xavierbol does this really matter? In 16.0 on top of that... I'm not even sure I would consider this as an imp in master.